README.md

```diff
@@ -28,7 +28,7 @@ The Ganttilla draws roadmap based on the `JSON` descriptor files in the assets d
 
 ### Gitlab
```

Comment:
Some details what is used, and how the comments setting lanes and dates are working is needed.